src/transformers/models/esm/modeling_esm.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, dim: int):
         # Generate and save the inverse frequency buffer (non trainable)
         inv_freq = 1.0 / (10000 ** (torch.arange(0, dim, 2).float() / dim))
         inv_freq = inv_freq
-        self.register_buffer("inv_freq", inv_freq, persistent=False)
+        self.register_buffer("inv_freq", inv_freq)
```

Comment:
We don't need any
```python
_keys_to_ignore_on_load_xxx
```
after changing this line?

Other than this, LGTM!

Reply:
No, it wasn't ignored that's the whole point :-)